localization/english/wc_game_concepts_l_english.yml

```diff
@@ -0,0 +1,17 @@
+﻿l_english:
+ game_concept_union:0 "Union"
+ game_concept_unions:0 "Unions"
+ game_concept_union_possessive:0 "Union's"
```

Comment:
could we go without the possessive and use "of the Union" instead? Would make german (probably french and other languages too) more easy to translate...